edx_proctoring/templates/proctoring/seq_proctored_exam_submitted.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You have submitted this proctored exam for review
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your proctoring session results: <b> Pending </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      Proctoring session results are usually available 24-48 hours after you submit your exam.
```

Comment:
@chrisndodge Please add an extra sentence before what we currently have:
"After you have ended the proctoring session, the recorded data is uploaded for review. "
(Followed by "Proctoring session results are usually available...")